### ISSUE TITLE
@@create no longer exists

### ISSUE THREAD

**Issue by dead-claudia** [lukehoban/es6features#43]
[Relevant discussion](https://github.com/rwaldron/tc39-notes/blob/master/es6/2014-09/sept-24.md#object-instantiation-redo).

**Comment by impinball** [comment]
@lukehoban Ping

**Comment by lukehoban** [comment]
Yes - this section needs to be modified.  But I think it does need to address the new model, and my understanding is that that is not yet 100% finalized.  I'm hoping that we can update this section in the next few days once the final decision on the class instantiation model is finalized in the draft spec.

**Comment by webbedspace** [comment]
"the next few days"... :/

Anyway, the actual bones of subclassing now looks like this:
* instantation of all exotic behaviour is done in the parent constructor
* `this` is unusable in subclasses' constructors until after `super()` is called.

**Comment by impinball** [comment]
@lukehoban ping...
